tests/test_template_types.py

```diff
@@ -54,6 +54,47 @@ def test_govuk_banner(show_banner):
         assert "GOV.UK" not in str(email)
 
 
+def test_brand_banner_shows():
```

Comment:
I think this test would be better if it passed in `brand_logo`, `brand_name` and `brand_colour` but set `brand_banner=False`, and then assert that the logo, name and colour don’t appear in the HTML. A bit more solid than asserting for arbitrary HTML I think.